requirements/quality.txt

```diff
@@ -50,9 +50,9 @@ code-annotations==1.3.0
     #   -r requirements/test.txt
     #   edx-lint
     #   edx-toggles
-confluent-kafka[avro,schema-registry]==1.9.2
+confluent-kafka[avro,schema-registry]==2.0.2
```

Comment:
None of the confluent-kafka 2 changes would be likely to affect us:

- https://github.com/confluentinc/confluent-kafka-python/blob/master/CHANGELOG.md#v202
- https://github.com/confluentinc/librdkafka/releases/tag/v2.0.0